c/driver/postgresql/statement.cc

```diff
@@ -683,6 +693,17 @@ AdbcStatusCode PostgresStatement::ExecuteQuery(struct ArrowArrayStream* stream,
       return na_res;
     }
 
+    // If there are no output columns (e.g. a CREATE or UPDATE), then
+    // don't use COPY (which would fail anyways)
+    if (root_type.n_children() == 0) {
+      RAISE_ADBC(ExecuteUpdateQuery(rows_affected, error));
+      struct ArrowSchema schema;
+      std::memset(&schema, 0, sizeof(schema));
```

Comment:
Is there any consideration to using memset versus `{0}` during assignment? I think I've seen the latter more often in the code base

Reply:
Mostly just a bad habit of mine since it's consistent between C/C++; {0} is perfectly fine too